ui-test-util/jsonFiles/v7/choice_all.json

```diff
@@ -0,0 +1,261 @@
+{
+  "ccpa":{
+    "signedLspa":false,
+    "dateCreated":"2022-10-16T12:57:57.264Z",
+    "cookies":[
```

Comment:
@Nevazhnovu we should ignore `signedLspa`, `cookies`, `actions` from both `ccpa` and `gdpr` objects. It's not used in the SDKs so less code to look at :)